addons/website/static/src/snippets/s_website_form/000.js

```diff
@@ -113,7 +113,10 @@ odoo.define('website.s_website_form', function (require) {
                     if (!$field.val() && _.has(dataForValues, field) && dataForValues[field]) {
                         $field.val(dataForValues[field]);
                     } else if (!$field.val() && $field.data('fill-with')) {
-                        $field.val(self.preFillValues[$field.data('fill-with')]);
+                        var value = self.preFillValues[$field.data('fill-with')];
```

Comment:
```suggestion
                        const value = self.preFillValues[$field.data('fill-with')];
```

Reply:
or 

`$field.val(self.preFillValues[$field.data('fill-with')] || '')`

Reply:
@julien-banken Can you consider this one? one-liner without the need of a variable